### ISSUE TITLE
[BLUS30504] Crashes without displaying unless set at 480p

### ISSUE THREAD

**Issue by John-Gee** [RPCS3/rpcs3#3837]
Hello,

I'm using the rpcs build rpcs3-v0.0.4-2017-12-02-bfb3144-9055_linux64.bin from the site (building my own changed nothing in this).

When starting BLUS30504/Ken's Rage at the recommanded 720p the game will crash with this log:
https://gist.github.com/John-Gee/9136f276995f660e6353aed9ca169c5b

It is the same with my default 1080p.

I have tried the various CPU options, and tried both OGL and Vulkan with no difference either. After rereading the log many times, because of this line "E {rsx::thread} RSX: NV3089_IMAGE_IN_SIZE: Invalid blit dimensions passed" I tried the first resolution in the list of supported resolutions, 480p, and TADA! the game runs (slowly and with some visual glitches but those are unrelated.).

I am running Arch Linux 64b, using a i7-4790k and a R280X on mesa-git (I doubt the build number matters as I've had that issue ever since RPCS3 started playing my other games on Linux, but in case I'm on 98124.cfc5af588c).

I'm happy to give more details or try anything to offer more information.

Thank you!

**Comment by kd-11** [comment]
Try running with PPU interpreter. There was no crash, the program did not like something about the environment it was running in and cleanly exited by itself.

**Comment by John-Gee** [comment]
I just tried with the 2nd interpreter, fast, and it was the same.

A window opened up saying it was loading loading the precompiled shaders and then died. Once I cleared the shaders it died even quicker. Switching back to 480p works.

**Comment by John-Gee** [comment]
So after looking at #4906 I realized people could still get in Ken's Rage 2, whereas I've never been able to and so I diffed our logs.

Turns out the issue for that game, and for the one this issue was created, comes from the Aspect Ratio. If I let it be auto, which I always did, Ken's Rage 1 requires the exact resolution of 480p to boot and Ken's Rage 2 will never boot no matter what I pick, but if I set the Aspect Ratio to 16/9 things change a bit: KR1 runs at 720p, on top of 480p, and KR2 boots at 720p, but still not at 576p.

I don't have any game that is not 16/9 so I guess leaving it on that is a fine workaround for me, but if feasible it may be a nice fix. If you think it's simple enough, I wouldn't mind some pointers into where I should look to write a patch myself.

(Unrelated but I used to be able to see what was happening in RPCS3's log window, but now it's all fairly empty apart from the 2 system lines and I need to look at the log files, what happened?)

Thanks!
